.travis.yml

```diff
@@ -1,20 +1,21 @@
 language: go
-sudo: false
+
 go:
-  - 1.7
-  - 1.8
+  - 1.9
```

Comment:
better if we use 1.9.x?

Reply:
nevermind, it's the same https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use :+1:

Reply:
1.9 is the latest release in the 1.9 minor version (that is, 1.9.x), so it's actually the same